### DISCUSSION TITLE
publish a new version

### DISCUSSION THREAD

**Discussion by IsraelZeromski** [slowli/jwt-compact#145] [General]
Hi @slowli, 
is possible publish a new version with x5t jwt headers changes?
Or when are you planned publish a new version of lib?.

**Comment by slowli** [comment]
Hi! Would a pre-release be fine for your purposes? I've planned a pre-release first at the end of this week (Friday or Saturday).

**Comment by slowli** [comment]
FYI, I've published [the 0.8.0-beta.1 release](https://crates.io/crates/jwt-compact/0.8.0-beta.1).

**Reply by IsraelZeromski** [reply]
thank you so much!
